<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1042</width>
    <height>562</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>180</width>
     <height>225</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>225</height>
    </size>
   </property>
   <property name="title">
    <string>系统设置</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leHostaddr">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>端口：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePort">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>客户端ID：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leClientId">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>用 户 名：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leUsernam">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>密    码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePassword">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>心跳(秒)：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leKeepAlive">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pbConnectToHost">
        <property name="text">
         <string>连接</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>387</y>
     <width>180</width>
     <height>314</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="title">
    <string>发布主题</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_7">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>主题：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lePublicTopic"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>服务质量：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbxQosPublish"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTextEdit" name="tePublicData"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QPushButton" name="pbClearPublish">
          <property name="text">
           <string>清除数据</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbPublishTopic">
          <property name="text">
           <string>发布</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>231</y>
     <width>180</width>
     <height>150</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>180</width>
     <height>150</height>
    </size>
   </property>
   <property name="title">
    <string>订阅主题</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_5">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>主题：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="leSubscribeTopic"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>服务质量：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="cbxQosSubscribe"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QPushButton" name="pbClearRecv">
            <property name="text">
             <string>清除接收</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pcSubscribeTopic">
            <property name="text">
             <string>订阅</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="cbxSubscribedTopic"/>
          </item>
          <item>
           <widget class="QPushButton" name="pbUnsubscribeTopic">
            <property name="text">
             <string>取消订阅</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>186</x>
     <y>0</y>
     <width>730</width>
     <height>701</height>
    </rect>
   </property>
   <property name="title">
    <string>接收区</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <widget class="QTextEdit" name="teRecvData"/>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
